### PR TITLE
AI Studio Bug Fixes

### PR DESCRIPTION
**Pull request by Abdulhuq**
- Corrected wrong url of `getSchemaNameUrl()`
- `getNlqData()` code fixed
- Added example in readme file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -152,6 +152,27 @@ getChartData();
...
...e/Float","semantic_type":null,"fingerprint":{"global":{"distinct-count":18,"nil%":0},"type":{"type/Number":{"min":13,"q1":16.121320343559642,"q3":32.75,"max":50,"sd":11.397075648311427,"avg":24.68}}}},{"display_name":"Low","field_ref":["field","Low",{"base-type":"type/Float"}],"name":"Low","base_type":"type/Float","effective_type":"type/Float","semantic_type":null,"fingerprint":{"global":{"distinct-count":16,"nil%":0},"type":{"type/Number":{"min":11,"q1":12.694841350859116,"q3":26,"max":43,"sd":9.481912605939092,"avg":19.64}}}}]},"insights":[{"previous-value":18,"unit":"hour","offset":554173.141530699,"last-change":-0.16666666666666666,"col":"Average","slope":-27.692307691919787,"last-value":15,"best-fit":["+",5488330.3818478165,["*",-554148.5544930139,["log","x"]]]},{"previous-value":14,"unit":"hour","offset":429669.413840075,"last-change":-0.14285714285714285,"col":"Low","slope":-21.470769230465464,"last-value":12,"best-fit":["+",4255285.115637468,["*",-429649.75721279817,["log","x"]]]}],"questionId":1370,"answerId":1152,"entityId":7193,"entityType":"CHART"},"questionId":1370,"extraData":{"userId":121873,"apiKeyId":88,"apiType":3,"date":"2024-10-15T00:00:00.000Z"},"nlqLogId":49}}
```
Comment (hitesh23k) [inline README.md:172]: We can add only basic output structure/blueprint without actual data

File: README.md [hunk 2]
```diff
@@ -152,6 +152,27 @@ getChartData();
...
 ```
+
+Here's an example to get the data from AI studio of DappLooker
+
+```jsx
+//Typescript
+import { DappLookerNlqAPI } from "dapplooker-sdk";
+
+async function getResults(): Promise<any> {
+  let apiKey = "d6fbd65055ef4510807fa4eb1662792b"
```
Comment (hitesh23k) [inline README.md:163]: Don't expose API key here

File: typescript/src/ai-studio/NlqConstants.ts [hunk 3]
```diff
@@ -6,7 +6,7 @@ class NLQConstants {
     }
 
     public get getSchemaNameUrl(): string {
-        return `https://api.dapplooker.com/project`;
+        return `https://api.dapplooker.com/projects`;
```
Comment (hitesh23k) [inline typescript/src/ai-studio/NlqConstants.ts:9]: It this change relevant to current fix?

File: README.md (hunk 2 above)
Comment (Abdulhuq) [reply README.md:163]: Done.

File: README.md (hunk 1 above)
Comment (Abdulhuq) [reply README.md:172]: Added a basic blueprint instead of whole data.

**Review by realchoubey — APPROVED** [review]

**Pull request merged**